src/NzbDrone.Core/Indexers/Definitions/GreatPosterWall.cs

```diff
@@ -186,6 +191,20 @@ protected string GetDownloadUrl(int torrentId, bool canUseToken)
 
         return url.FullUri;
     }
+
+    protected virtual List<IndexerCategory> ParseCategories(GreatPosterWallTorrent torrent)
+    {
+        var cats = new List<IndexerCategory>();
+
+        cats.Add(torrent.Resolution switch
+        {
+            var res when _hdResolutions.Contains(res) => NewznabStandardCategory.MoviesHD,
```

Comment:
Are we able to add similar category capabilities to searching (like Avistaz). If we are returning categories for these results it would be have these categories in the indexers capabilities categories list and be able to search with them.